web/src/components/CityContentLayout.tsx

```diff
@@ -35,8 +36,13 @@ const CityContentLayout = (props: CityContentLayoutProps): ReactElement => {
     disableScrollingSafari = false,
     showFooter = true,
     city,
+    embedded,
   } = props
 
+  if (embedded) {
+    return <>{children}</>
+  }
+
```

Comment:
What is this used for?

Reply:
I've thought about moving this logic to the CitzContent Layout, to remove duplicate code. But as the offers tab will be removed in the future, all offers would also be "embedded", therefore I reverted this change again.